source/devices/AM62LX/linux/Linux_Performance_Guide.rst

```diff
@@ -21,16 +21,486 @@ Table:  Evaluation Modules
+    "lat_ctx-2-128k (us)","14.58"
+    "lat_ctx-2-256k (us)","31.29"
+    "lat_ctx-4-128k (us)","13.82"
+    "lat_ctx-4-256k (us)","19.00"
+    "lat_fs-0k (num_files)","217.00"
+    "lat_fs-10k (num_files)","105.00"
+    "lat_fs-1k (num_files)","152.00"
+    "lat_fs-4k (num_files)","134.00"
+    "lat_mem_rd-stride128-sz1000k (ns)","50.64"
+    "lat_mem_rd-stride128-sz125k (ns)","6.21"
+    "lat_mem_rd-stride128-sz250k (ns)","13.20"
+    "lat_mem_rd-stride128-sz31k (ns)","3.90"
+    "lat_mem_rd-stride128-sz50 (ns)","2.41"
+    "lat_mem_rd-stride128-sz500k (ns)","44.90"
+    "lat_mem_rd-stride128-sz62k (ns)","4.92"
+    "lat_mmap-1m (us)","57.00"
+    "lat_ops-double-add (ns)","3.21"
+    "lat_ops-double-div (ns)","17.65"
+    "lat_ops-double-mul (ns)","3.21"
+    "lat_ops-float-add (ns)","3.21"
+    "lat_ops-float-div (ns)","10.43"
+    "lat_ops-float-mul (ns)","3.21"
+    "lat_ops-int-add (ns)","0.80"
+    "lat_ops-int-bit (ns)","0.54"
+    "lat_ops-int-div (ns)","4.81"
+    "lat_ops-int-mod (ns)","5.08"
+    "lat_ops-int-mul (ns)","3.45"
+    "lat_ops-int64-add (ns)","0.80"
+    "lat_ops-int64-bit (ns)","0.54"
+    "lat_ops-int64-div (ns)","7.62"
+    "lat_ops-int64-mod (ns)","5.88"
+    "lat_ops-int64-mul (ns)","3.98"
+    "lat_pagefault (us)","0.95"
+    "lat_pipe (us)","22.06"
+    "lat_proc-exec (us)","1411.25"
+    "lat_proc-fork (us)","1221.80"
+    "lat_proc-proccall (us)","0.01"
+    "lat_select (us)","37.59"
+    "lat_sem (us)","3.63"
+    "lat_sig-catch (us)","5.82"
+    "lat_sig-install (us)","0.79"
+    "lat_sig-prot (us)","0.95"
+    "lat_syscall-fstat (us)","2.00"
+    "lat_syscall-null (us)","0.51"
+    "lat_syscall-open (us)","555.70"
+    "lat_syscall-read (us)","0.78"
+    "lat_syscall-stat (us)","5.07"
+    "lat_syscall-write (us)","0.75"
+    "lat_tcp (us)","1.02"
+    "lat_unix (us)","27.34"
+    "latency_for_0.50_mb_block_size (nanosec)","44.90"
+    "latency_for_1.00_mb_block_size (nanosec)","25.32 (min 0.00, max 50.64)"
+    "pipe_bandwidth (MBs)","399.08"
+    "pipe_latency (microsec)","22.06"
+    "procedure_call (microsec)","0.01"
+    "select_on_200_tcp_fds (microsec)","37.59"
+    "semaphore_latency (microsec)","3.63"
+    "signal_handler_latency (microsec)","0.79"
+    "signal_handler_overhead (microsec)","5.82"
+    "tcp_ip_connection_cost_to_localhost (microsec)","59.62"
+    "tcp_latency_using_localhost (microsec)","1.02"
+
+Dhrystone
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Dhrystone is a core only benchmark that runs from warm L1 caches in all
+modern processors. It scales linearly with clock speed.
+
+Please take note, different run may produce different slightly results.
+This is advised to run this test multiple times in order to get maximum
+performance numbers.
+
+Execute the benchmark with the following:
+
+::
+
+    runDhrystone
+
+.. csv-table:: Dhrystone Benchmarks
+    :header: "Benchmarks","am62lxx_evm-fs: perf"
+
+    "cpu_clock (MHz)","400.00"
+    "dhrystone_per_mhz (DMIPS/MHz)","9.20"
```

Comment:
something seems off here? 400MHz? I would expect a DMIPS around 13 as it's the same as all other K3 A53s